packages/daimo-api/src/db/db.ts

```diff
@@ -60,6 +63,16 @@ export class DB {
           CREATE TABLE IF NOT EXISTS paymaster_whitelist (
             name VARCHAR(32) PRIMARY KEY
           );
+
+          CREATE TABLE IF NOT EXISTS invitegraph (
```

Comment:
nit: invite_graph

Reply:
i'd also consider adding a created_date column / can default to now()

Reply:
done both